demo/redirect-flow-example/src/App.tsx

```diff
@@ -568,7 +568,11 @@ function App() {
         </button>
 
         <button onClick={async () => uiConsole(await coreKitInstance._UNSAFE_exportTssKey())} className="card">
-          [CAUTION] Export TSS Private Key
+          [CAUTION] Export TSS Private Key for SECP256k1
+        </button>
+
+        <button onClick={async () => uiConsole(await coreKitInstance._UNSAFE_exportTssEd25519Seed())} className="card">
+          [CAUTION] Export TSS Seed for ED25519
```

Comment:
The demo only ever supports one key type. No need to create two buttons. You could alter the button text and functionality based on `tssLib.keyType`?